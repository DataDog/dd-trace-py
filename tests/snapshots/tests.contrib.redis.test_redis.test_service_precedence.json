[[
  {
    "name": "redis.command",
    "service": "env-redis",
    "resource": "GET cheese",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "4489d8d8a85846dfb4125bdf108bb45c"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "out.port": 6379,
      "out.redis_db": 0,
      "redis.args_length": 2,
      "system.pid": 786
    },
    "duration": 4036300,
    "start": 1632158528924457700
  }]]
