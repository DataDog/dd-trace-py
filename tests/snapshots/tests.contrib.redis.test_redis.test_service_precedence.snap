[[{"name" "redis.command"
   "service" "env-redis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1596062189185636000
   "duration" 2615000
   "meta" {"runtime-id" "5b043ff215454b3685a99fa16242038e"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 76424
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 2}}]]
